### PR TITLE
SAPI-464-Makefile update

### PR DESCRIPTION
**Pull request by bpathak-ch**
Updates for
    * Makefile
    * Start.sh

### CODE REVIEW

**Review by dlgroves-ch — APPROVED** [review]

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    wget -O "${APP_DIR}/private_env" "${CONFIG_URL}/${ENVIRONMENT}/private_env"
+    wget -O "${APP_DIR}/global_env" "${CONFIG_URL}/${ENVIRONMENT}/global_env"
+    wget -O "${APP_DIR}/app_env" "${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}/env"
+    source "${APP_DIR}/private_env"
+    source "${APP_DIR}/global_env"
+    source "${APP_DIR}/app_env"
+fi
+
+# Read brokers from environment and split on comma
+IFS=',' read -ra BROKERS <<< "${KAFKA_STREAMING_BROKER_ADDR}"
```
Comment (dlgroves-ch) [inline start.sh]: Remove this logic; no argument called `-cache-broker-url` is defined.

**Review by dlgroves-ch — CHANGES_REQUESTED** [review]
No routes.yaml file.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+dist: clean build package
+
+.PHONY: xunit-tests
+xunit-tests: GO111MODULE=off
+xunit-tests:
+	go get github.com/tebeka/go2xunit
+	@set -a; go test -v $(TESTS) -run 'Unit' | go2xunit -output $(xunit_output)
+
+.PHONY: lint
+lint: GO111MODULE=off
```
Comment (bpathak-ch) [inline Makefile:67]: need to remove this line as GO111MODULE is already configured as ON in line12

Comment (dlgroves-ch) [reply Makefile:67]: No we don't; golint does not support gomodules.

Comment (bpathak-ch) [reply Makefile:67]: changes reverted back

**Review by dlgroves-ch — CHANGES_REQUESTED** [review]
Still unnecessary logic around setting the cache broker URL in the start script.

**Review by dlgroves-ch — APPROVED** [review]

**Pull request merged**